### PR TITLE
Add image_url and description to PoolCreated event

### PR DESCRIPTION
**Pull request by ahzab**

### CODE REVIEW

File: programs/degen-pools/src/pool_options.rs [hunk 1]
```diff
@@ -8,6 +8,12 @@ pub struct PoolOption {
     pub title: String,
     pub value: u64,
 }
+
+#[event]
+pub struct OptionCreated {
+    pub pool_account: Pubkey,
```
Comment (AngSin) [inline programs/degen-pools/src/pool_options.rs:14]: Let's include the option title in the event so the backend can store it also

File: tests/claimWin.test.ts [hunk 2]
```diff
@@ -7,107 +7,117 @@ import {expect} from "chai";
 import {program} from "./utils/constants";
 
 describe('Wins claiming', () => {
-   it('should not let a user claim twice', async () => {
```
Comment (AngSin) [inline tests/claimWin.test.ts:10]: curious, diff looks weird, was a lot changed here?

Comment (ahzab) [reply tests/claimWin.test.ts:10]: I'm just reformatted the file, I'll revert the formatting changes.

Comment (ahzab) [reply tests/claimWin.test.ts:10]: Done

File: programs/degen-pools/src/pool_options.rs (hunk 1 above)
Comment (ahzab) [reply programs/degen-pools/src/pool_options.rs:14]: Done

**Pull request merged**